backend/app/controllers/eventsController/db/onOrderChange.js

```diff
@@ -1,38 +1,38 @@
-const { Orders } = require("../../../db/models");
+const { Orders } = require('../../../db/models');
 
 module.exports = async (mongoEvent, io) => {
-    if (mongoEvent.operationType === "insert") {
-        const ownerRoomName = `${mongoEvent.fullDocument.restaurantId}:owner`;
-        const tabRoomName = `${mongoEvent.fullDocument.restaurantId}:${mongoEvent.fullDocument.tabId}`;
-        //todo remove duplicate code from here. make a function that returns 
-        //the order and use it here and in controllers
-        const order = await Orders.findById(mongoEvent.fullDocument._id)
-        .populate('items')
-        .populate({
-          path: 'tabId',
-          populate: {
-            path: 'tableId'
-          }
-        }).exec();
+	if (mongoEvent.operationType === 'insert') {
+		const ownerRoomName = `${mongoEvent.fullDocument.restaurantId}:owner`;
+		const tabRoomName = `${mongoEvent.fullDocument.restaurantId}:${mongoEvent.fullDocument.tabId}`;
+		//todo remove duplicate code from here. make a function that returns 
+		//the order and use it here and in controllers
+		const order = await Orders.findById(mongoEvent.fullDocument._id)
+			.populate('items')
+			.populate({
+				path: 'tabId',
+				populate: {
+					path: 'tableId'
+				}
+			}).exec();
 
-        io.to(ownerRoomName).emit("order:new", { order });
-        console.log(`New order sent in room: "${ownerRoomName}"`);
+		io.to(ownerRoomName).emit('order:new', { order });
+		console.log(`New order sent in room: "${ownerRoomName}"`);
 
-        io.to(tabRoomName).emit("order:new", { order });
-        console.log(`New order sent in room: "${tabRoomName}"`);
-    }
+		io.to(tabRoomName).emit('order:new', { order });
+		console.log(`New order sent in room: "${tabRoomName}"`);
+	}
 
-    if (mongoEvent.operationType === "update") {
-        const order = await Orders.findById(mongoEvent.documentKey._id);
-        const ownerRoomName = `${order.restaurantId}:owner`;
-        const tabRoomName = `${order.restaurantId}:${order.tabId}`;
+	if (mongoEvent.operationType === 'update') {
+		const order = await Orders.findById(mongoEvent.documentKey._id);
+		const ownerRoomName = `${order.restaurantId}:owner`;
+		const tabRoomName = `${order.restaurantId}:${order.tabId}`;
         
-        io.to(ownerRoomName).emit("order:update", { id: order._id, status: order.status });
-        console.log(`New order update sent in room: "${ownerRoomName}"`);
+		io.to(ownerRoomName).emit('order:update', { id: order._id, status: order.status });
+		console.log(`New order update sent in room: "${ownerRoomName}"`);
 
-        io.to(tabRoomName).emit("order:update", { id: order._id, status: order.status });
-        console.log(`New order update sent in room: "${tabRoomName}"`);
+		io.to(tabRoomName).emit('order:update', { id: order._id, status: order.status });
+		console.log(`New order update sent in room: "${tabRoomName}"`);
         
 
-    }
+	}
 };
\ No newline at end of file
```

Comment:
Overall, the code patch seems to be functioning correctly. Here are some suggestions for improvement:

1) In both insert and update conditions, it's good practice to include an else block to handle unexpected operation types.

2) In the insert condition, the todo message suggesting that a function should be created to remove duplicate code is appropriate. This would improve readability and reduce potential bugs if this functionality is used in multiple places.

3) In the update condition, it might be better to use the updated document rather than retrieving it from the database again with Orders.findById(mongoEvent.documentKey._id). This could save a significant performance cost in situations where there are many updates.

4) It may be beneficial to include error handling code to gracefully handle database and socket connection errors.

5) The lack of a newline at the end of the file may cause issues in some text editors. Adding a newline at the end of the file can avoid potential problems.